vmdk_plugin/vmdkops/esx_vmdkcmd.go

```diff
@@ -85,19 +86,30 @@ func (vmdkCmd EsxVmdkCmd) Run(cmd string, name string, opts map[string]string) (
 	defer C.free(unsafe.Pointer(ans))
 
 	// connect, send command, get reply, disconnect - all in one shot
-	_, err = C.Vmci_GetReply(C.int(EsxPort), cmdS, beS, ans)
-
-	if err != nil {
-		var errno syscall.Errno
-		errno = err.(syscall.Errno)
-		msg := fmt.Sprintf("'%s' failed: %v (errno=%d).", cmd, err, int(errno))
-		if errno == syscall.ECONNRESET {
-			msg += " Hit communication issue with ESX (vmci or ESX service)\n"
-			msg += " Please refer to the FAQ https://github.com/vmware/docker-volume-vsphere/wiki#faq"
+	retryCount := 10
```

Comment:
For example, plugin server "always" tries to attach the volume provided. And errors out if the volume is attached. So, if the first attach was OK and a network error happened the client is going to redo the attach and this time the server sends back an error which the client receives and interprets wrongly (having already attached the disk).

Pls. identify errors that are transient and push upwards and let the upper layer (plugin.go) decide on getting current status of the volume and decide to retry.